src/order_book/response.rs

```diff
@@ -0,0 +1,92 @@
+use derive_getters::Getters;
+use serde::{Deserialize, Serialize};
+
+use crate::models::Response;
+
+/// Represents the response from the Horizon server when querying for the details of an order book.
+/// 
+/// This struct represents the response from the Horizon server when querying for the details of an order book.
+/// It includes the bids, asks, base, and counter fields.
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize, Getters)]
+#[serde(rename_all = "camelCase")]
+pub struct DetailsResponse {
+    /// The bids in the order book.
+    pub bids: Vec<Bid>,
+    /// The asks in the order book.
+    pub asks: Vec<Ask>,
+    /// The base asset of the order book.
+    pub base: Base,
+    /// The counter asset of the order book.
+    pub counter: Counter,
+}
+
+/// Represents a bid in the order book.
+/// 
+/// This struct represents a bid in the order book. It includes the price_r, price, and amount fields.
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize, Getters)]
+#[serde(rename_all = "camelCase")]
+pub struct Bid {
+    #[serde(rename = "price_r")]
+    pub price_r: PriceR,
```

Comment:
Weet jij wat een `price_r` is? Kunnen we die hier toelichten? Zelfde voor `price` en `amount`